test/registration/test_correspondence_rejectors.cpp

```diff
@@ -130,8 +130,9 @@ TEST (CorrespondenceRejectors, CorrespondenceRejectionPoly)
    * but not too many
    */
   EXPECT_GE(accepted_frac, ground_truth_frac);
-  EXPECT_LE(accepted_frac, 1.5f*ground_truth_frac);
-  
+  // Factor 1.5 raised to 1.6 see #2995 for details
```

Comment:
Maybe mention the reason directly (to account for variance in single and double precision across diff implementations) along with referring to the issue